components/automate-ui/src/app/pages/project/rules/project-rules.component.scss

```diff
@@ -16,9 +16,16 @@ chef-page {
     overflow-y: scroll;
   }
 
-  chef-input {
+  input {
     height: 45px;
     width: 100%;
+    outline: 0;
+  }
+
+  chef-error {
+    background-color: #DC267F;
+    color: #FFFFFF;
+    border-radius: 0 0 4px 4px;
```

Comment:
`:ng-deep` from the chefInput Directive/styling isn't reaching deep enough when the error is sectioned inside an extra div, so I ported the style over here.  This extra div is needed to guarantee that the error will show above the helper text.